### PR TITLE
DIABLO-274, scheduling with Kaltura API: get start and end date/times right

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-274

Kaltura integration work is _far_ from done but the schedule-recordings API calls are getting through. This PR fixes start/end params and, by removing the `KALTURA_SCHEDULE_API_AVAILABLE` config, opens this integration up to proper QA testing.

*Bonus:* Error alert emails go to `EMAIL_SYSTEM_ERRORS`, not the admin user email.

### CODE REVIEW

File: diablo/externals/kaltura.py [hunk 1]
```diff
@@ -76,110 +76,114 @@ def schedule_recording(
             room,
             term_id,
     ):
```
Comment (johncrossman) [inline diablo/externals/kaltura.py:78]: I recommend `?w=1` when reviewing diff below.

File: tests/test_lib/test_kaltura_util.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        with override_config(app, 'CURRENT_TERM_BEGIN', _get_wednesdaay_august_26()):
+            first_day_start = get_first_matching_datetime_of_term(
+                meeting_days=['TU'],
+                time_hours=9,
+                time_minutes=15,
+            )
+            assert first_day_start == datetime(2020, 9, 1, 9, 15)
+
+
+def _get_wednesdaay_august_26():
```
Comment (lyttam) [inline tests/test_lib/test_kaltura_util.py:75]: @johncrossman Typo in method name?

Comment (johncrossman) [reply tests/test_lib/test_kaltura_util.py:75]: Yup :) Amending.

**Comment by johncrossman** [comment]
@lyttam The Travis build finished and is green: https://travis-ci.com/github/ets-berkeley-edu/diablo/builds/165250898. Github did not get the memo. I think this is ready for review/merge.

**Pull request merged**